Project.toml

```diff
@@ -3,19 +3,18 @@ uuid = "7d188eb4-7ad8-530c-ae41-71a32a6d4692"
 version = "0.3.0"
 
 [deps]
-BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
 HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
+ZipFile = "a5390f91-8eb1-5f08-bee0-b1d1ffed6cea"
 
 [compat]
-BinaryProvider = "0.5"
 HTTP = "0.8"
 JSON = "0.21"
+ZipFile = "~0.8"
```

Comment:
If possible, please include ZipFile v0.9, since that version switched to JLL binary dependencies.

Reply:
Done. I'm keeping 0.8 because we need to retain Julia 1.0 support for JuMP.